### PR TITLE
add Abuse to experimental

### PR DESCRIPTION
**Pull request by proyvind**
This updated version of abuse works properly without X (in contrast to dead upstream)

### CODE REVIEW

File: scriptmodules/ports/abuse.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="abuse"
+rp_module_desc="Abuse"
+rp_module_license="GPL https://raw.githubusercontent.com/Xenoveritas/abuse/master/COPYING"
+rp_module_section="exp"
+
+# abuse-lib & abuse-sfx will pull in the older abuse package which only works under X
+# no children or animals harmed though, so whatever..
```
Comment (joolswills) [inline scriptmodules/ports/abuse.sh]: Please remove these comments and I will merge. thanks.

Comment (joolswills) [reply scriptmodules/ports/abuse.sh]: (the comment regarding the packages can be added just above the getDepends, but the other comment can be removed.)

**Comment by proyvind** [comment]
done

**Comment by joolswills** [comment]
thanks.

**Pull request merged**